pygmt/helpers/decorators.py

```diff
@@ -169,14 +169,31 @@
             documentation is at :gmt-docs:`gmt.html#l-full`.""",
     "n": r"""
         interpolation : str
-            [**b**\|\ **c**\|\ **l**\|\ **n**][**+a**][**+b**\ *BC*][**+c**][**+t**\ *threshold*].
-            Select interpolation mode for grids. You can select the type of
-            spline used:
-
-            - **b** for B-spline
-            - **c** for bicubic [Default]
-            - **l** for bilinear
-            - **n** for nearest-neighbor""",
+            [**b**\|\ **c**\|\ **l**\|\ **n**][**+a**][**+b**\ *BC*][**+c**]\
+            [**+t**\ *threshold*].
+            Select interpolation mode for grids.
+
+            - **b** to use B-spline smoothing.
+            - **c** to use bicubic interpolation.
+            - **l** to use bilinear interpolation.
+            - **n** to use nearest-neighbor value (for example to plot
+            categorical data).
+
+            The following modifiers are supported:
+
+            - **+a** to switch off antialiasing (where supported) [default uses
```

Comment:
```suggestion
            - **+a** to switch off antialiasing (where supported) [Default uses
```
To be consistent with the stylte @meghanrjones used in #1407 😉.